rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -422,6 +425,39 @@ public void registerBtcTransaction(
         }
     }
 
+    private void updateSvpFundTransactionValuesIfPossible(BtcTransaction transaction) {
+        Optional<Sha256Hash> svpFundTransactionHashUnsigned = provider.getSvpFundTxHashUnsigned();
+        if (!svpFundTransactionHashUnsigned.isPresent()) {
+            return;
+        }
+
+        if (!isTheSvpFundTransaction(svpFundTransactionHashUnsigned.get(), transaction)) {
+            return;
+        }
+
+        updateSvpFundTransactionValues(transaction);
+    }
```

Comment:
You can take advantage of the Optional API and make this more concise and functional as follows:

```java
private void updateSvpFundTransactionValuesIfPossible(BtcTransaction transaction) {
    provider.getSvpFundTxHashUnsigned()
        .filter(hash -> isTheSvpFundTransaction(hash, transaction))
        .ifPresent(hash -> updateSvpFundTransactionValues(transaction));
}
````

Reply:
nice! thanks

Reply:
updated :)